src/test/java/ai/elimu/rest/v2/crowdsource/WordContributionRestControllerTest.java

```diff
@@ -0,0 +1,32 @@
+package ai.elimu.rest.v2.crowdsource;
+
+import ai.elimu.util.JsonLoader;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Test;
+import selenium.DomainHelper;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.assertThat;
+
+public class WordContributionRestControllerTest {
+
+    private Logger logger = LogManager.getLogger();
+
+    @Test
+    public void testGetWordDataForCrowdSourcing() {
+        String jsonResponse = JsonLoader.loadJson(DomainHelper.getRestUrlV2() + "/crowdsource/word-contributions/word-data");
+        logger.info("word-dataResponse: " + jsonResponse);
+
+        JSONArray allophoneJSONArray = new JSONArray(jsonResponse);
+        logger.info("wordsJSONArray.length(): " + allophoneJSONArray.length());
+        assertThat(allophoneJSONArray.length() > 0, is(true));
+
+        JSONObject allophoneJsonObject = allophoneJSONArray.getJSONObject(0);
+        assertThat(allophoneJsonObject.getLong("id"), not(nullValue()));
+        assertThat(allophoneJsonObject.getString("valueIpa"), not(nullValue()));
+    }
+
```

Comment:
Can we replace this with a unit test that tests the `POST` request?

Reply:
Yes, this logic should be moved to a separate class because the retrieval of LetterToAllophoneMappings will be handled by a separate Controller. And this should then be replaced by a POST request.

Reply:
@nya-elimuai I think it's best if we handle the POST request Tests in a separate Issue because we do not currently have any tests for POST requests. For now, I will leave the class without any tests and move the GET request test to a seperate class What do you think?

Reply:
@UmenR Yes, handling this in a separate issue sounds good to me. Could you create a new GitHub issue for this, please? - https://github.com/elimu-ai/webapp/projects/7